AGS02MA.cpp

```diff
@@ -301,6 +318,15 @@ bool AGS02MA::_writeRegister(uint8_t reg)
   return (_error == 0);
 }
 
+uint16_t AGS02MA::_dataMSB(uint8_t * buf)
```

Comment:
unclear from function name it is a get

Reply:
True, `_buffer` is local. Though I was mimicking the `_CRC8` method here, allowing any arbitrary buffer to be used.
No huge preferences though, it's such a small function body it might as well be inlined.

Reply:
Renamed and using local variable.